spec/features/management/account_spec.rb

```diff
@@ -29,4 +29,91 @@
     expect(user.reload.erase_reason).to eq "Deleted by manager: manager_user_#{Manager.last.user_id}"
   end
 
-end
\ No newline at end of file
+  scenario "Send reset password email to currently managed user session" do
+    user = create(:user, :level_three)
+    login_managed_user(user)
+    visit management_account_path
+
+    click_link 'Reset password'
+
+    expect(page).to have_content 'Edit user account: Reset password'
+
+    click_link 'Send reset password email'
+
+    expect(page).to have_content 'Email correctly sent.'
+
+    logout
+
+    sent_token = /.*reset_password_token=(.*)".*/.match(ActionMailer::Base.deliveries.last.body.to_s)[1]
```

Comment:
I wouldn't test here that the reset password form works, that's already tested at https://github.com/AyuntamientoMadrid/consul/blob/master/spec/features/users_auth_spec.rb#L333-L353 instead the objective of this scenario is to check that the email is sent